File: app/src/main/java/test/platzi/com/raian/com/org/simpledynamit/model/city/Result.java

package test.platzi.com.raian.com.org.simpledynamit.model.city;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Result {

    @SerializedName("city")
    @Expose
    private String city;
    @SerializedName("country")
    @Expose
    private String country;
    @SerializedName("locations")
    @Expose
    private int locations;
    @SerializedName("count")
    @Expose
    private int count;

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }

    public int getLocations() {
        return locations;
    }

    public void setLocations(int locations) {
        this.locations = locations;
    }

    public int getCount() {
        return count;
    }

    public void setCount(int count) {
        this.count = count;
    }

}

File: app/src/main/java/test/platzi/com/raian/com/org/simpledynamit/model/city/Result.kt
package test.platzi.com.raian.com.org.simpledynamit.model.city

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Result {

    @SerializedName("city")
    @Expose
    var city: String? = null
    @SerializedName("country")
    @Expose
    var country: String? = null
    @SerializedName("locations")
    @Expose
    var locations: Int = 0
    @SerializedName("count")
    @Expose
    var count: Int = 0

}
